### ISSUE TITLE
runbot, couldn't confirm sale order

### ISSUE THREAD

**Issue by fanha99** [OCA/stock-logistics-warehouse#544]
hi, I'm checking the modules in this repository.
when I try to confirm a sale order, I got the error as below.

```
  File "/.repo_requirements/odoo/odoo/http.py", line 651, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/.repo_requirements/odoo/odoo/http.py", line 310, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/.repo_requirements/odoo/odoo/tools/pycompat.py", line 87, in reraise
    raise value
  File "/.repo_requirements/odoo/odoo/http.py", line 693, in dispatch
    result = self._call_function(**self.params)
  File "/.repo_requirements/odoo/odoo/http.py", line 342, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/service/model.py", line 97, in wrapper
    return f(dbname, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/http.py", line 335, in checked_call
    result = self.endpoint(*a, **kw)
  File "/.repo_requirements/odoo/odoo/http.py", line 937, in __call__
    return self.method(*args, **kw)
  File "/.repo_requirements/odoo/odoo/http.py", line 515, in response_wrap
    response = f(*args, **kw)
  File "/home/odoo/OCB-11.0/addons/web/controllers/main.py", line 938, in call_button
    action = self._call_kw(model, method, args, {})
  File "/home/odoo/OCB-11.0/addons/web/controllers/main.py", line 926, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/.repo_requirements/odoo/odoo/api.py", line 689, in call_kw
    return call_kw_multi(method, model, args, kwargs)
  File "/.repo_requirements/odoo/odoo/api.py", line 680, in call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/home/odoo/OCB-11.0/addons/sale/models/sale.py", line 565, in action_confirm
    self._action_confirm()
  File "/home/odoo/OCB-11.0/addons/sale_stock/models/sale_order.py", line 40, in _action_confirm
    order.order_line._action_launch_procurement_rule()
  File "/home/odoo/OCB-11.0/addons/procurement_jit/sale.py", line 12, in _action_launch_procurement_rule
    res = super(SaleOrderLine, self)._action_launch_procurement_rule()
  File "/home/odoo/OCB-11.0/addons/sale_stock/models/sale_order.py", line 287, in _action_launch_procurement_rule
    self.env['procurement.group'].run(line.product_id, product_qty, procurement_uom, line.order_id.partner_shipping_id.property_stock_customer, line.name, line.order_id.name, values)
  File "/home/odoo/build/OCA/stock-logistics-warehouse/stock_orderpoint_uom/models/procurement_group.py", line 23, in run
    name, origin, values)
  File "/home/odoo/OCB-11.0/addons/stock/models/procurement.py", line 186, in run
    getattr(rule, '_run_%s' % rule.action)(product_id, product_qty, product_uom, location_id, name, origin, values)
  File "/home/odoo/OCB-11.0/addons/stock/models/procurement.py", line 78, in _run_move
    data = self._get_stock_move_values(product_id, product_qty, product_uom, location_id, name, origin, values, group_id)
  File "/home/odoo/build/OCA/stock-logistics-warehouse/stock_secondary_unit/models/procurement_rule.py", line 18, in _get_stock_move_values
    if sale_line.secondary_uom_id:
AttributeError: 'sale.order.line' object has no attribute 'secondary_uom_id'
```

**Comment by rousseldenis** [comment]
@pedrobaeza @sergio-teruel I think this is for you

**Comment by pedrobaeza** [comment]
OK, I see the problem... telling my colleague for fixing it.

**Comment by pedrobaeza** [comment]
Fixed in #545, so closing.
